src/agent/onefuzz-task/src/tasks/coverage/generic.rs

```diff
@@ -174,7 +175,7 @@ impl CoverageTask {
             context.save_and_sync_coverage().await?;
         }
 
-        info!("report coverage");
+        info!("report final coverage");
```

Comment:
this is not the final coverage.
the task can still report coverage in the loop that reads from the queue.